pkg/wmi/wmi.go

```diff
@@ -53,3 +55,20 @@ func CheckOptionalFeatureStatus(feature string) (FeatureInstallState, error) {
 
 	return dst[0].InstallState, nil
 }
+
+func QueryWithContext(timeout time.Duration, query string, dst interface{}, connectServerArgs ...interface{}) error {
```

Comment:
This function doesn't really make sense.
- if we use a `context`, we should pass it as an argument but not create the context inside the function.
- If the only usage of the `context` is the timeout, check `time.After`